exporter/influxdbexporter/README.md

```diff
@@ -0,0 +1,141 @@
+* `bucket` (Required): Bucket is the InfluxDB bucket name that telemetry will be written to.
+* `protocol` (Optional): Protocol is the method used to deliver metrics to InfluxDB. Must be one of `line-protocol`, `grpc`. Defaults to `line-protocol`.
+* `line_protocol_server_url` (Required): LineProtocolServerUrl is the destination for Protocol `line-protocol`.
+* `line_protocol_auth_token`: LineProtocolAuthToken is the authentication token for Protocol `line-protocol`.
+* `timeout` (Optional): Timeout is the timeout for every attempt to send data to InfluxDB. Defaults to `5s`.
+
+
+Example:
+```yaml
+exporters:
+  influxdb:
+    org: "my-org"
+    bucket: "my-bucket"
+    protocol: "line-protocol"
+    line_protocol_server_url: "http://localhost:8086"
+    line_protocol_auth_token: "my-token"
+    timeout: 10s
+```
+
+The full list of settings exposed for this exporter are documented [here](config.go).
+
+## Schema
+
+InfluxDB/IOx is an open-source time series storage engine, capable of storing and querying large (and small) time series datasets.
+The schema used by this OpenTelemetry exporter is optimized for InfluxDB/IOx performance.
+
+### Definitions
+
+An OpenTelemetry ["Signal"](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/overview.md#opentelemetry-client-architecture) is general term for instances of categories of telemetry.
+For example:
+- Trace spans
+- Metrics
+- Logs
+
+An OpenTelemetry Signal ["Attribute"](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/common.md#attributes) to be a key-value pair whose key is a non-null, non-empty string, and whose value is not null and of type string, boolean, float64, int64, or homogeneous arrays of the previously mentioned primitives.
+Attribute names should follow the [naming guidelines](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-and-label-naming.md).
+For example:
+- `service.name`
+- `host.name`
+- `cloud.region`
+- `org_id`
+- `env`
+
+An OpenTelemetry Signal Label is similar to a Signal Attribute.
+
+An OpenTelemetry Signal ["Resource"](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md) Attribute is "an immutable representation of the entity producing telemetry".
+In other words, the entity that generated the Signal.
+For example:
+- Host machine or VM
+- Kubernetes node, pod, and container
+- Application
+
+An OpenTelemetry Signal ["Instrumentation Library"](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/overview.md#instrumentation-libraries) Attribute is "a library that enables OpenTelemetry observability for another library".
+In other words, the library used by the host application to generate the Signal.
+For example:
+- The Go OpenTelemetry implementation
+- The Rust OpenTelemetry implementation
```

Comment:
This is not a good example. Instrumentation libraries instrument other _specific_ libraries (e.g. "cassandra" instrumentation library). Instrumentation libraries are not generic language libraries (that would be an _OpenTelemetry SDK_) like this example implies.

Reply:
I'll simply drop this example.